### PR TITLE
Inform framework about added/removed views

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

**Comment by goderbauer** [comment]
cc @loic-sharma @dkwingsmt

File: lib/ui/platform_dispatcher.dart [hunk 1]
```diff
@@ -274,13 +274,15 @@ class PlatformDispatcher {
     assert(!_views.containsKey(id));
     _views[id as int] = _createView(id);
     _viewConfigurations[id] = const _ViewConfiguration();
+    _invoke(onMetricsChanged, _onMetricsChangedZone);
```
Comment (loic-sharma) [inline lib/ui/platform_dispatcher.dart:278]: I'm fine with doing this as a temporary hack, but really the embedder API's `FlutterAddViewInfo` should have a view metrics member and `FlutterEngineAddView` should send window metrics. /cc @dkwingsmt

See: https://docs.google.com/document/d/1hgb3FsHRNJmZYvSGi6uQXrek3KwmGCcI24fUilV-mfI/edit#heading=h.v5bj81fmonnl

Comment (goderbauer) [reply lib/ui/platform_dispatcher.dart:278]: Agreed.

Comment (goderbauer) [reply lib/ui/platform_dispatcher.dart:278]: Left a TODO so we don't forget. Are you ok with submitting this then? Would make experimenting with the prototype a little easier...

**Review by loic-sharma — APPROVED** [review]

**Comment by loic-sharma** [comment]
Apologies for the delay on merging!

**Pull request merged**